### ISSUE TITLE
Bugfix: add support for multiple pods per node

### ISSUE THREAD

**Issue by razvan** [stackabletech/zookeeper-operator#133]
## The problem
Currently only one ZooKeeper pod per node is supported.

## Acceptance criteria
- there are integration tests for this case.

**Comment by lfrancke** [comment]
This will be fixed as part of the config reorg, correct?
(which includes a move to roles and role groups?

**Comment by maltesander** [comment]
Yes but you need another rolegroup + selector with different config properties for ports etc.
